subscription-service/src/main/kotlin/com/egm/stellio/subscription/service/SubscriptionService.kt

```diff
@@ -254,14 +257,15 @@ class SubscriptionService(
             """
             UPDATE geometry_query
             SET georel = :georel, geometry = :geometry, coordinates = :coordinates,
-                pgis_geometry = ST_GeomFromText(:wkt_coordinates) 
+                pgis_geometry = ST_GeomFromText(:wkt_coordinates), geoproperty= :geoproperty
             WHERE subscription_id = :subscription_id
             """
         )
             .bind("georel", geoQuery["georel"] as String)
             .bind("geometry", geoQuery["geometry"] as String)
             .bind("coordinates", storedCoordinates)
             .bind("wkt_coordinates", wktCoordinates)
+            .bind("geoproperty", geoQuery["geoproperty"] as String)
```

Comment:
will it work if `geoQuery["geoproperty"]` is null?

Reply:
geoproperty can't be null because location are used by default

Reply:
you are in an update here (and you are not dealing with a `GeoQuery` object but with a map)

Reply:
Ok It's good I modified this point